docs/node/tutorial.mdx

```diff
@@ -151,6 +152,58 @@ Current validator set:
   1. {pubkey = a2d8e307117695c8d0d3adbda136cb2ad630c43b8672a0eba46d79501831b794, power = 1}
 ```
 
+## Deploying A Database
+
+To deploy a database to the network, we will need to locate the network chain ID and any of the node's HTTP listen addresses.
+
+### Getting The Chain ID
+
+Use the `kwil-admin node status` command from earlier to get the chain ID:
+
+```bash
+$ kwil-admin node status --rpcserver unix:///tmp/node0.sock
```

Comment:
Here I'm wondering why an operator would not just look at their genesis.json file.  Unlike settings in config.toml, `chain_id` has no default and cannot be left blank.  The `setup init` or `setup testnet` commands will generate a new chain ID and write it to the genesis.json file(s).

Reply:
Noobs (e.g. me) get nervous / confused when you stop giving them commands in a tutorial. For the average noob, the mental journey of being told to look in my genesis.json for the chain id would be something like:
1. Wait, where are those files? *_nervousness begins_
2. Ohh, I guess kwil-admin created a bunch of node files? *_clicks on testnet/node0_
3. Oh shit, there are are a lot of files here *_nervousness intensifies_
4. Chain id is probably in this config.toml file right? Oh wait no, the instructions say gensis.json. *_sheepish feeling_
5. OH wow, genesis.json is a big file. Where is chain id? *_at this point, sweating bullets_
6. Okay, i think it is this item on line 3, but I am not entirely sure. *_unconfidently continues with tutorial, now doubting my self-worth and the meaning of life._

Obviously a bit dramatic haha. But I think it is useful if, in a tutorial, we provide a command and an expected output. The user can then run that command, see if they get the same expected output, and once they do, find the information they need and proceed confidently. Printing the chain id when a new network is setup could also be a solution, as we discussed earlier.

Reply:
Sorta fair, but for sake of argument:

```
  grep chain_id ~/.kwild/genesis.json 
  "chain_id": "kwil-chain-Np2caxD3",
```

Reply:
That is even better!